rewrite-java/src/main/java/org/openrewrite/java/internal/template/BlockStatementTemplateGenerator.java

```diff
@@ -406,16 +412,17 @@ private void template(Cursor cursor, J prior, StringBuilder before, StringBuilde
             before.insert(0, ev.getName());
         } else if (j instanceof J.EnumValueSet) {
             after.append(";");
+        } else if (j instanceof Expression && j == prior) {
+            before.insert(0, "__M__.any(");
+            after.append(");");
         }
         template(next(cursor), j, before, after, templated);
     }
 
     private void insertControlWithBlock(J body, StringBuilder before, StringBuilder after, Runnable insertion) {
```

Comment:
Not sure why this never caused any issues, but since all the called `insertion` runnables do an `insert(0, ...)` I think the order was wrong here.